cl/corpus_importer/tasks.py

```diff
@@ -988,10 +990,21 @@ def get_pacer_case_id_and_title(
         docket_number,
         court_id,
     )
-    if not cookies:
-        # Get cookies from Redis if not provided
-        cookies = get_pacer_cookie_from_cache(user_pk)  # type: ignore
-    s = ProxyPacerSession(cookies=cookies)
+
+    if cookies_data:
+        cookies, proxy_address = cookies_data
+    elif user_pk:
+        cookies_from_cache = get_pacer_cookie_from_cache(user_pk)
+        if isinstance(cookies_from_cache, tuple):
+            cookies, proxy_address = cookies_from_cache
+        cookies, proxy_address = cookies_from_cache, settings.EGRESS_PROXY_HOST
+    else:
+        # If the user_pk is unavailable, cookies cannot be retrieved from the
+        # cache.
+        self.request.chain = None
```

Comment:
Is it better to throw an exception here? In theory we should never arrive at this code, right?

Reply:
> In theory we should never arrive at this code, right?

You're right! I'll update the code to throw an exception.